### PR TITLE
Allen Hai - Responsive-Web-Design

### PR DESCRIPTION
**Pull request by coetry**
Still need to finish, but getting there

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -64,7 +68,7 @@ h1, h2, h3, h4, h5 {
 }
 
 .container {
-    width: 880px;
+    width: 68.75%;
```
Comment (mixelpixel) [inline css/index.css]: @coetry

Comment (mixelpixel) [inline css/index.css]: @coetry

**Comment by mixelpixel** [comment]
@coetry Looks like you are comfortable with Git and the GitHub Pull Request system - love that your commits _**tell the story of your software engineering adventure!**_ 🎉 

If you want to add more detailed information, try adding a body to your commit messagesx. For example:

```
git commit -m "message with a closing quotation mark

You can hit enter twice to put a new line in between your commit title and a body!
- you can add bullet points
- explain in detail
- what you did
- and this helps your teammates
- and improves your own analysis
- best of all, after you've left the project for awhile
- these comments will help you reorient to your work!
- So don't forget that closing quotation mark :)"
```

**Comment by coetry** [comment]
@mixelpixel sounds good!

**Pull request closed without merging**